draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -243,6 +243,25 @@ Any candidates exposed to the application via local descriptions MUST be
 identical to those provided during candidate gathering (i.e., MUST NOT
 contain private host IP addresses).
 
+#### mDNS registration and query answer
```

Comment:
This is essentially a more detailed version of step 4 above. I suggest simply expanding that part of the algorithm with the details below on registration. The parts on query response can stay in implementation guidance, but I would move this section to the top of the implementation guidance.

Reply:
ok